Marker/Marker/Utility/Extensions/TextView/UITextViewExtension.swift

```diff
@@ -29,11 +29,7 @@ public extension UITextView {
     ///   - textStyle: Text style object containing style information.
     func setMarkdownText(_ markdownText: String, using textStyle: TextStyle) {
         if let linkColor = textStyle.linkColor {
-            #if swift(>=4.0)
```

Comment:
Why did you remove this statement? Are we dropping Swift versions older than 4.0 in this PR as well? If that's the case, we should at least update the README for the minimum required versions of Swift and Xcode.

Reply:
It's the same for both old and new

Reply:
@ThibaultKlein @malinkas86 if I'm not mistaken, we are implicitly dropping older Swift versions with some of these changes already (e.g.: `UIApplication.LaunchOptionsKey`). Even 4.0 and 4.1 I think are dropped with this?

Reply:
Better example (since the above only affects the example project): `NSAttributedString.Key`

Reply:
Got it, as long as it's reflected in the README I'm fine with that. It should say at the top of the README what version of Swift is compatible with the current version.

Reply:
Hm the supported Swift version is currently not mentioned in the Readme at all as far as I see. I think it could be for sure if you make that addition. Conversely there's also a `spec.swift_version = '4.2'` we could add in the podspec instead and highlight the upgrade in the [release notes](https://github.com/prolificinteractive/Marker/releases) for this upgrade.

Reply:
@malinkas86 once the Swift supported version is documented in the spec file and README I'm good with this comment.

Reply:
Done.